todo_app/templates/index.html

```diff
@@ -11,8 +11,22 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+          <li>{{ item.title }}</li>
+        {% endfor %}
       </ul>
     </div>
   </div>
+
+    <div class = "form_collect">
+        <form action = "/add_items" method = "post" class = "form">
```

Comment:
You should use `url_for` here too:

```suggestion
        <form action = "{{ url_for("create_items") }}" method = "post" class = "form">
```

Reply:
Note the argument for the url_for method is the name of the route function.

Reply:
Noted, will change, thank you!